doc/doxygen/src/getting-started.md

```diff
@@ -140,3 +140,15 @@ make -C examples/gnrc_networking/ term \
     TERMPROG=gtkterm \
     TERMFLAGS="-s 115200 -p /dev/ttyACM0 -e"
 ~~~~~~~~
+
+Configuring an application                         {#configuring-an-application}
+--------------------------
+Many modules in RIOT offer configuration options in compile-time.
```

Comment:
configuration options that will be considered during compile-time.

Reply:
Changed.